file-formats/dras/examples/z_aff_example_dras.dras.json

```
@@ -0,0 +1,8 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example CDS Aspect",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
```

```suggestion
    "originalLanguage": "en"
```

`"abapLanguageVersion": "standard"` is typically not rendered, because it is the default value.